packages/webapp/src/members/components/member-card.tsx

```diff
@@ -13,22 +13,73 @@ interface Props {
 
 export const MemberCard = ({ member }: Props) => {
     return (
-        <div className="max-w-md bg-white rounded-lg p-8 flex flex-col w-full mt-10 md:mt-0 shadow-md">
-            <Heading size={2} className="mb-4">
-                {member.name}
-            </Heading>
+        <div className="px-2 sm:px-8 flex flex-col max-w-xl">
             <MemberSocialLinks member={member} />
-            <Text className="mt-4">{member.bio}</Text>
+            <section className="py-4">
+                <MemberBio bio={member.bio} />
+            </section>
             <div className="mx-auto">
-                <Button
-                    href={`${ipfsBaseUrl}/${member.inductionVideo}`}
-                    target="_blank"
-                    className="mt-10 inline-flex"
+                <SocialButton
+                    handle="View induction ceremony"
                     icon={FaVideo}
-                >
-                    Induction Ceremony
-                </Button>
+                    color="black"
+                    size={5}
+                    href={`${ipfsBaseUrl}/${member.inductionVideo}`}
+                />
             </div>
         </div>
     );
 };
+
+const MemberBio = ({ bio }: { bio: string }) => {
+    const [expanded, setExpanded] = useState(false);
```

Comment:
The UI doesn't quite look right when the bio is longer than 300 or so chars...it's a lot of text. This collapses it and makes it easy to toggle expanded state.

Reply:
now you are showing off